wee_slack.py

```diff
@@ -2587,7 +2587,11 @@ def unfurl_ref(ref, ignore_alt_text=False):
                 display_text = ref.split('|')[1]
             else:
                 url, desc = ref.split('|', 1)
-                display_text = "{} ({})".format(url, desc)
+                if (config.unfurl_prevent_auto_linking and
+                        re.match(r"^\w+:(//)?{}$".format(desc), url)):
```

Comment:
Maybe use `re.escape(desc)` to ensure regex metacharacters aren't treated specially?

Reply:
Definitely, should have thought of that.